src/Calculator.ts

```diff
@@ -240,6 +239,9 @@ export function calculateMeasureReports(
       report.subject = subjectReference;
     }
 
+    // add supplemental data elements to contained and as evaluatedResource references
+    addSDE(report, report.measure, result);
```

Comment:
I guess we should only do this if `options.calculateSDEs` is true

Reply:
great catch!